docs/sql/statements/update.rst

```diff
@@ -63,7 +63,7 @@ Parameters
 :output_expression:
     An expression to be computed and returned by the UPDATE command after each
     row is updated. The expression can use any column names of the table or
-    ``*`` to return all columns.
+    ``*`` to return all columns. Also system columns can be returned.
```

Comment:
Good idea, to mentioned it explicitly. It would also possible to put the information in the description like we did on the `insert` https://github.com/crate/crate/pull/9615/files#diff-e3fc8f9cdc5c6f2d07f3c38d2e7d5045R49

Reply:
yup, agreed

the "also" implies that returning system columns is optional. but my understanding is that `*` will return all columns _including_ system columns. is the case? if so, I would probably phrase along those lines

Reply:
`*` returns indeed also the returning the system columns, and joe is right. It is mentioned nowhere so far here.

Reply:
`*` doesn't resolve to system columns per default:

```
cr> update tbl set x = x + 1 returning *;
+---+
| x |
+---+
| 2 |
+---+
UPDATE 1 row in set (0.068 sec)
```

They have to be included explicitly:

```
cr> update tbl set x = x + 1 returning *, _id, _seq_no;
+---+----------------------+---------+
| x | _id                  | _seq_no |
+---+----------------------+---------+
| 3 | qJtfDHAB5yNd4FVQW2sZ |       2 |
+---+----------------------+---------+
UPDATE 1 row in set (0.012 sec)
```

We should keep it that way to mirror the behavior of `SELECT *`

Reply:
Thanks @mfussenegger for pointing this out correctly, i was wrong here. So then the addition to the documentation in the output expression section from @joemoe makes total sense to me. Is there any  other opinion here ?

Reply:
Can I get an `ack` on that?